src/scenes/home/benefit/benefit.js

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import Section from 'shared/components/section/section';
 import LinkButton from 'shared/components/linkButton/linkButton';
 import Sabio from '../../../images/codeSchoolLogos/sabio.png';
-import CodeFellows from '../../../images/codeSchoolLogos/cf-logo-horizontal-2-color-black.png';
+import CodeFellows from '../../../images/codeSchoolLogos/codefellows_small.png';
 import Nodejs from '../../../images/NodeSummitSF_ColorLogoWeb.jpg';
 import srvd from '../../../images/srvd.png';
 import ebay from '../../../images/320px-EBay_logo.png';
 import secureSet from '../../../images/SecureSet.logoUSE.jpg';
 import PVA from '../../../images/pvaoregon.jpg';
+import PXD from '../../../images/pdxcglogo.svg';
```

Comment:
Typo: PXD --> PDX

Let's also be more specific with all the logo imports

"OrgNameLogo" would probably best for all above and going forward. It's okay if the variable name is long.